build.gradle

```diff
@@ -33,6 +33,7 @@ buildscript {
 plugins {
     id 'com.jfrog.bintray' version '1.7.3'
     id 'org.unbroken-dome.test-sets' version '1.4.2'
+    id 'com.gradle.plugin-publish' version '0.12.0' apply false
```

Comment:
Unfortunately, I didn't go deep on this one and I'd rather prefer to leave like this for the moment.

When I was trying to come up with a nice sub-module structure, I couldn't make it work with the `plugins` declaration clause within the `gradle-plugin/build.gradle`. I didn't go further on the analysis, so I kept consistency with what we had already (which was working). But your logic is correct: it would be nice to have project-specific things inside each module's `build.gradle` files, indeed.....

Reply:
I will resolve this conversation for now.